frontend/src/Components/HomePage/UpcomingTrips/Upcoming.jsx

```diff
@@ -25,6 +30,21 @@ function Upcoming() {
         fetchTrips();
     }, []);
 
+    const moveToCompleted = async (trip) => {
```

Comment:
Can we include this change in separate PR? Fetching recommended flights LGTM, but I'd like us to chat about this part more carefully. I don't think updating flight status on frontend behind the scenes is the way to go.

Reply:
On it!